lib/internal/modules/esm/load.js

```diff
@@ -128,7 +129,7 @@ async function defaultLoad(url, context = kEmptyObject) {
       context = { __proto__: context, source };
     }
 
-    if (format == null) {
+    if (!format) {
```

Comment:
unrelated change

Reply:
I think this is related, as IMO comparing format to `null` (even loosely) gives the impression that it should be null at a point. I think the `!format` is a better comparison.

Reply:
Wouldn't `!format` imply that it should be a boolean? I disagree it's a better comparison, if it is an empty string you'd get a false positive